cmake/findDependencies.cmake

```diff
@@ -40,10 +40,14 @@ if (NOT USE_MATCHCOMPILER_OPT MATCHES "Off")
 endif()
 
 if (NOT USE_BUNDLED_TINYXML2)
-    find_package(tinyxml2)
+    find_package(tinyxml2 QUIET)
     if (NOT tinyxml2_FOUND)
-        message(FATAL_ERROR "tinyxml2 has not been found")
-    else()
-        message(STATUS "Found tinyxml2 (version ${tinyxml2_VERSION})")
+        find_library(tinyxml2_LIBRARY tinyxml2)
```

Comment:
please use the `REQUIRED` option.

Reply:
The `REQUIRED` option has been added to `find_library()` in cmake 3.18
https://cmake.org/cmake/help/v3.18/release/3.18.html#commands

Reply:
Oh - sorry about that. I didn't check the documentation since I was quite confident it had been there for a while. The usual bad assumption when it comes to CMake...